.github/workflows/master.yml

```diff
@@ -567,7 +567,7 @@ jobs:
       - name: Install container networking plugins
         shell: bash
         run: |
-          CNI_PLUGIN_VERSION="v1.5.1"
+          CNI_PLUGIN_VERSION=\"v1.7.1\"
```

Comment:
Why do we need to quote this now and it was not needed before?

Reply:
We have several instances of this - if this is needed it should be in all version, if not, it should be removed. Maybe the issue is using double qoute instead of single quote? It is not the same in github actions yaml.

Reply:
I think this is fixed by https://github.com/kubernetes/minikube/pull/20769